packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -38,6 +38,17 @@ class CodePushNotFoundException extends CodePushException {
   CodePushNotFoundException({required super.message, super.details});
 }
 
+class _CodePushClientWrapper extends http.BaseClient {
```

Comment:
Nit: doc explaining the purpose of this wrapper

Reply:
Yup will do